receiver/snmpreceiver/integration_test.go

```diff
@@ -25,6 +26,8 @@ import (
 )
 
 func TestIntegration(t *testing.T) {
+	t.Skip("Broken test, see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34489")
```

Comment:
Looks like this PR will close https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34489, could you file a new issue for this test?

Reply:
Updated